components/LayerObject.vue

```diff
@@ -56,57 +61,110 @@
 </script>
 
 <template>
-    <ul class="layers">
+    <div class="empty-display" v-if="arrayZ.length === 0">
+        <img src="/public/Barista explaining6.png" alt="" />
+        <div class="empty-display_text">
+            <h1>No layers to display</h1>
+            <p>Start by adding an asset to the canvas.</p>
+            <button class="canvas-btn">Canvas</button>
+        </div>
+    </div>
+    <ul class="layers" :style="{ 'margin-top': navHeightPx }">
         <li
             v-for="(element, index) in arrayZSorted"
             :key="element.id"
             class="layer"
             :accessKey="element.id"
             @click="selectLayer(element.id, index)"
-            :style="{ border: index === selection ? '1px solid black' : 'none' }"
+            :style="{ border: index === selection ? `3px solid ${$primary}` : `1px solid ${$primary}` }"
         >
             <div class="asset-image">
                 <img class="img" :src="element.src" :alt="element.alt" />
             </div>
             <p class="layer-text">{{ element.alt }}</p>
             <div class="chevrons">
-                <div
-                    class="expand-less icon"
+                <button
+                    class="expand-less icon icon-btn"
                     :style="{ opacity: index > 0 ? 1 : 0.25, cursor: index > 0 ? 'pointer' : 'not-allowed' }"
```

Comment:
rather do this over classes.

Reply:
Since it is a button you could also dynamically set the disabled state, which properly would be best.